python/ray/data/context.py

```diff
@@ -91,13 +94,16 @@ def get_current() -> "DatasetContext":
                     optimize_fuse_shuffle_stages=DEFAULT_OPTIMIZE_FUSE_SHUFFLE_STAGES,
                     actor_prefetcher_enabled=DEFAULT_ACTOR_PREFETCHER_ENABLED,
                     use_push_based_shuffle=DEFAULT_USE_PUSH_BASED_SHUFFLE,
+                    scheduling_strategy=None,
```

Comment:
Should the default value being "HYBRID" so by default we won't inherit parent's pg?

Reply:
Ah good catch, I mistakenly thought that `None` would result in the hybrid scheduling policy, not inheriting parent's placement groups! I'll change this to`"DEFAULT"`.